apps/src/netsim/NetSimRouterNode.js

```diff
@@ -342,6 +347,22 @@ NetSimRouterNode.prototype.stopSimulation = function () {
 };
 
 /**
+ * Puts the router into the given DNS mode, triggers a remote update,
+ * and creates/destroys the network's automatic DNS node.
+ * @param {DnsMode} newDnsMode
+ */
+NetSimRouterNode.prototype.setDnsMode = function (newDnsMode) {
+  if (this.dnsMode === newDnsMode) {
+    return;
+  }
+
+  // TODO (bbuchanan): Handle DNS node management here
```

Comment:
I don't think so; this and the other TODO I've added in the PR are really captured in [#89655666](https://www.pivotaltracker.com/story/show/89655666), continuing work on the auto-dns feature.